xml/Makefile.am

```diff
@@ -69,7 +69,16 @@ API_request_base	= command-output	\
 CIB_cfg_base		= options nodes resources constraints fencing acls tags alerts
 
 # Names of all schemas (including top level and those included by others)
-API_base		= $(API_request_base) fence-event failure generic-list item node-attrs node-history nodes resources status
+API_base		= $(API_request_base)	\
+			  fence-event failure	\
```

Comment:
did you mean to put these on separate lines?

Reply:
Yes, thought I caught all of those, thanks